etmain/characters/temperate/axis/fieldops.char

```diff
@@ -1,15 +1,15 @@
 characterDef
 {
-	skin					"fieldops"
+	skin	"fieldops"
 
-	mesh					"models/players/temperate/axis/fieldops/body.mdm"
```

Comment:
This change seems outside of scope of this issue?

Reply:
It should be fixed now. I must've been half-asleep given much of this was in a rush.

Reply:
No worries! ;)